package.json

```diff
@@ -53,16 +53,15 @@
     "walk-sync": "^0.3.1"
   },
   "dependencies": {
-    "amd-name-resolver": "^0.0.6",
     "broccoli-plugin": "^1.3.0",
     "exists-sync": "^0.0.4",
     "fs-tree-diff": "^0.5.2",
     "heimdalljs": "^0.2.1",
     "heimdalljs-logger": "^0.1.7",
     "mkdirp": "^0.5.1",
+    "mr-dep-walk": "^1.0.1",
```

Comment:
Haha, okay, it's a fine name, just wanted to make sure I wasn't missing anything